DQM/SiStripMonitorCluster/python/SiStripMonitorCluster_cfi.py

```diff
@@ -249,6 +291,26 @@
         xmin = cms.double(-0.5)
         ),
 
+     TProfNClusStrip = cms.PSet(
+        Nbinsx = cms.int32(3565),
+        xmin = cms.double(0),
+        xmax = cms.double(3564),
+        Nbinsy = cms.int32(200),
+        ymin = cms.double(-0.5),
+        ymax = cms.double(99999.5),
+        globalswitchon = cms.bool(True)
+        ),
+
+     TProfNClusPixel = cms.PSet(
+        Nbinsx = cms.int32(3565),
+        xmin = cms.double(0),
+        xmax = cms.double(3564),
+        Nbinsy = cms.int32(200),
```

Comment:
Only two instances. We need to monitor single bunch crossings so the x 
axis bins can not be modified. Maybe we can reduce the y bins to half 
(i.e 100) but the current value is an optimal one used also in offline 
analysis. Let us know if this is a big issue.

Il 11/08/2016 15.42, Slava Krutelyov ha scritto:
>
> In DQM/SiStripMonitorCluster/python/SiStripMonitorCluster_cfi.py 
> <https://github.com/cms-sw/cmssw/pull/15422#discussion_r74423562>:
>
> > @@ -249,6 +291,26 @@
> >          xmin = cms.double(-0.5)
> >          ),
> >
> > +     TProfNClusStrip = cms.PSet(
> > +        Nbinsx = cms.int32(3565),
> > +        xmin = cms.double(0),
> > +        xmax = cms.double(3564),
> > +        Nbinsy = cms.int32(200),
> > +        ymin = cms.double(-0.5),
> > +        ymax = cms.double(99999.5),
> > +        globalswitchon = cms.bool(True)
> > +        ),
> > +
> > +     TProfNClusPixel = cms.PSet(
> > +        Nbinsx = cms.int32(3565),
> > +        xmin = cms.double(0),
> > +        xmax = cms.double(3564),
> > +        Nbinsy = cms.int32(200),
>
> How many of instances of these will appear?
> 3565x200 ~1 M bins is already fairly large by itself.
> Will this run in PromptReco?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/cms-sw/cmssw/pull/15422/files/f9809ee7d33e765b8a898f678ee5acefd40e3f44#r74423562>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AKYUWkLC_hsapokMZvd2c51jQ1lUdcLLks5qeybHgaJpZM4Jh59Y>.
>